### PR TITLE
https://bugs.openmw.org/issues/253

### PR DESCRIPTION
**Pull request by lazydev2**
1. Dialogs in Russian version now works.
2. Destination names it travel window are now translated

### CODE REVIEW

**Comment by zinnschlag** [comment]
Can you please merge in master first? Your code uses at least one function that doesn't exist anymore (we just had a bit refactoring on the string handling).

File: apps/openmw/mwgui/dialogue.hpp [hunk 1]
```diff
@@ -109,6 +115,10 @@
         MyGUI::EditPtr     mDispositionText;
 
         PersuasionDialog mPersuasionDialog;
+
+        MWBase::WindowManager& mWindowManager;
```
Comment (ghost) [inline apps/openmw/mwgui/dialogue.hpp]: Duplicated member, already exists in WindowBase

**Comment by zinnschlag** [comment]
Also, completely redundant for a while now, since you can always get the WindowManager via MWBase::Environment.

**Comment by lazydev2** [comment]
Done

**Pull request merged**